### DISCUSSION TITLE
Home Assistent Add-On

### DISCUSSION THREAD

**Discussion by victor-vz** [mikebrady/shairport-sync#1654] [Ideas]
I have Home Assistant running on a Pi4. Will it be possible to expose the Aux to Airplay 2 with a simple Addon? I am not an expert and after loosing my mind with all the containers I try to run only Add-Ons

**Comment by mikebrady** [comment]
Thanks for the question. I'm afraid I don't have any experience using Shairport Sync with Home Assistant. Maybe the HA guys would have an answer...
